docs/source/index.rst

```diff
@@ -1,53 +1,164 @@
+management for any language. Here, you will find everything you need to get started
+using conda in your own projects.
+
+Install  :octicon:`download;1em;sd-text-primary`
+................................................
+
+We recommend the following methods to install conda:
+
+.. tab-set::
+
+    .. tab-item:: Windows :fab:`windows`
+
+        Miniconda installer for:
+
+        .. grid:: 2
+
+            .. grid-item::
+
+                .. button-link:: https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe
+                    :color: primary
+
+                    Windows x86 64-bit :octicon:`download`
+
+    .. tab-item:: MacOS :fab:`apple`
+
+        Miniconda installer for:
+
+        .. grid:: 2
+
+            .. grid-item::
+
+                .. button-link:: https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.pkg
+                    :color: primary
+
+                    MacOS x86 64-bit :octicon:`download`
+
+            .. grid-item::
+
+                .. button-link:: https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-arm64.pkg
+                    :color: primary
+
+                    MacOS M1 64-bit :octicon:`download`
+
+    .. tab-item:: Linux :fab:`linux`
+
+        Miniconda installer for:
+
+        .. grid:: 2
+
+            .. grid-item::
+
+                .. button-link:: https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+                    :color: primary
+
+                    Linux x86 64-bit :octicon:`download`
+
+    .. tab-item:: Homebrew :fa:`beer-mug-empty`
+
+        Run the following `Homebrew <https://brew.sh/>`_ command:
+
+        .. code-block:: bash
+
+            brew install miniconda
+
+    .. tab-item:: Chocolatey :octicon:`terminal` :fab:`windows`
+
+        Run the following `Chocolatey <https://chocolatey.org/>`_ command:
+
+        .. code-block:: bash
+
+            choco install miniconda3
+
+
+.. raw:: html
+
+    <p class="text-small">For more detailed instructions, see <a href="https://docs.conda.io/projects/miniconda/en/latest/">Miniconda's installation guide</a></p>
+
+New to conda? :octicon:`rocket;1em;sd-text-primary`
+...................................................
+
+If you are new to conda, we first recommend the following articles:
+
+.. grid:: 1 2 2 2
+    :gutter: 2
+
+    .. grid-item-card:: Getting started guide :octicon:`rocket;1em;sd-text-primary`
+        :link: /user-guide/getting-started
+        :link-type: doc
+
+        Learn the basics of using conda such as creating and adding packages to environments
+
+    .. grid-item-card:: Managing environments :octicon:`file-submodule;1em;sd-text-primary`
+        :link: /user-guide/tasks/manage-environments
+        :link-type: doc
+
+        Go in depth about environments and best practices for using them in your projects
+
+.. seealso::
+
+    Want to get even more in-depth training on how to use conda for free? Check out Anaconda's
```

Comment:
Got two "in depth"s here pretty close together. Maybe we could change the first one to "Go deeper with environments" or "Learn more about environments" or something similar.